cmd/diffoci/commands/diff/diff.go

```diff
@@ -66,6 +66,7 @@ func NewCommand() *cobra.Command {
 	flags.Bool("ignore-file-order", false, "Ignore file order in tar layers")
 	flags.Bool("ignore-file-mode-redundant-bits", false, "Ignore redundant bits of file mode")
 	flags.Bool("ignore-image-name", false, "Ignore image name annotation")
+	flags.Bool("canonical-paths", false, "Treat leading `./` `/` `` in file paths as canonical")
```

Comment:
Should this be called `ignore-non-canonical-path` for consistency with other flags?

Reply:
I am not sure.

Reply:
Maybe `--ignore-leading-slashes`?

Reply:
i believe it should be `--ignore-canonical-path-prefixes` if we insist on adding ignore as the flag prefix. However i like the --treat-canonical-paths-equally more..

Reply:
I don't have a strong preference, `--treat-canonical-paths-equally` is fine to me

Reply:
Done. chose `--treat-canonical-paths-equal`